Stack/Opc.Ua.Core/Stack/Tcp/ChannelQuotas.cs

```diff
@@ -120,6 +121,28 @@ public int MaxBufferSize
             }
         }
 
+        /// <summary>
+        /// The maximum number of secure channels
+        /// </summary>
+        public int MaxChannelCount
+        {
+            get
+            {
+                lock (m_lock)
```

Comment:
Here we do not expect any change to the m_maxChannelCount variable. Also, the lock is used in other unrelated places that would hamper performance under load.

Reply:
yes, I believe the locks in quotas can be removed, similar it was in servicemessagecontext.

Reply:
agreed, all locks in channelquotes are subject for removal, if they are just int or references, similar as in servicemessagecontext.